config.yaml.defaults

```diff
@@ -71,6 +73,10 @@ app:
       icon: GroupWork
       url: /groups
 
+    - name: Observing Runs
+      icon: LocalCafe
```

Comment:
Is this the right icon? Perhaps "Visibility" icon is more obvious?

Reply:
I appreciate Danny's sense of humor here, would leave it as is ;)